README.md

```diff
@@ -1,34 +1,24 @@
+![Logo](assets/logo/WastedApp_1.png)
```

Comment:
we should make it smaller, it takes up a lot of space on screen

Reply:
I made it smaller using html, check it now :)